source/solutions-library/fraud-detection-accelerator.txt

```diff
@@ -0,0 +1,211 @@
+.. _arch-center-is-fraud-detection-accelerator:
+
+===================================================
+Fraud Detection Accelerator Using AWS SageMaker
+===================================================
+
+.. meta:: 
+   :keywords: Fraud Detection, AWS SageMaker, Machine Learning, MongoDB Atlas, AI Solutions
+   :description: Revolutionize fraud detection in finance with MongoDB Atlas and Amazon SageMaker Canvas. Leverage real-time data and AI for stronger defenses against cybercrime.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+- **Use cases:** `Gen AI <https://www.mongodb.com/use-cases/artificial-intelligence>`__,
+  `Fraud Prevention
+  <https://www.mongodb.com/industries/financial-services/fraud-prevention>`__
+- **Industries:** `Financial Services
+  <https://www.mongodb.com/industries/financial-services>`__, `Insurance
+  <https://www.mongodb.com/industries/insurance>`__
+- **Products and tools:** `Atlas <https://www.mongodb.com/atlas/database>`__, `Atlas App
+  Services <https://www.mongodb.com/products/platform/atlas-app-services>`__, `Atlas
```

Comment:
The "Atlas App Services" link is broken and redirects back to "Products", but this is also true in the original article. [It looks like they're deprecated](https://www.mongodb.com/docs/atlas/app-services/). For migration purposes, the article looks good, but we might want to figure out some wording and put a disclaimer at the top that links to that page.

We could update the URL in this article to go to that docs page, so users can see it's deprecated. Same with the repeated URL at the end.